### PR TITLE
Animation

### PR DESCRIPTION
**Pull request by YasirHasn9**
# Description

Fixes # (issue) || User story it applies to

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Has This Been Tested?

- [ ] Yes
- - [ ] What was tested?
- [x] No

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed and commented by at least two peers
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/Home/Mission/Mission.js [hunk 1]
```diff
@@ -32,8 +32,14 @@ const Mission = props => {
           the students in the Bay Area.
         </p>
         <div>
-          <button>Pay It Forward</button>
-          <span>HOW WE ARE HELPING IMPROVE DESIGN INCLUSION</span>
+          <button>
+            <a href="#">Pay It Forward</a>
```
Comment (acarrillo3) [inline src/components/pages/Home/Mission/Mission.js:36]: the path for this one should be /donate

**Review by acarrillo3 — APPROVED** [review]
Look great!

**Review by DrakeAlia — DISMISSED** [review]

**Review by antonyk — APPROVED** [review]
nice

**Review by acarrillo3 — APPROVED** [review]
Looks great!

**Pull request merged**